databricks/koalas/generic.py

```diff
@@ -276,6 +276,30 @@ def var(self):
         """
         return self._reduce_for_stat_function(F.variance)
 
+    @property
+    def size(self) -> int:
+        """
+        Return an int representing the number of elements in this object.
+
+        Return the number of rows if Series. Otherwise return the number of
+        rows times number of columns if DataFrame.
+
+        See Also
+        --------
+        ndarray.size : Number of elements in the array.
```

Comment:
What will be this in the docs? Can this be linked?

Reply:
hmhm .... let me remove it. I think it's from numpy doc.